test/python/WMCore_t/Storage_t/SiteLocalConfig_t.py

```diff
@@ -156,6 +159,37 @@ def testLoadingConfigFromOverridenEnvVarriable(self):
                          "Error: Wrong site name.")
 
 
+    @attr("integration")
+    def testSlcPhedexNodesEqualPhedexApiNodes(self):
+        """
+        For each site, verify that the stageout node specified in
+        site-local-config.xml is the same as the one returned by the PhEDEx api.
+        """
+        os.environ["CMS_PATH"] = "/cvmfs/cms.cern.ch"
+
+        phedex = PhEDEx()
+        nodes = phedex.getNodeMap()["phedex"]["node"]
+
+        # Make a dict for translating the se names into regular site names.
+        node_map = {}
+        for node in nodes:
+            node_map[str(node[u"se"])] = str(node[str(u"name")])
+        
+        for d in os.listdir("/cvmfs/cms.cern.ch/SITECONF/"):
+            # Only T0_, T1_... folders are needed
+            if d[0] == "T":
+                os.environ['WMAGENT_SITE_CONFIG_OVERRIDE'] ='/cvmfs/cms.cern.ch/SITECONF/%s/JobConfig/site-local-config.xml' % (d)
+                try:
+                    slc = loadSiteLocalConfig()
+                except SiteConfigError as e:
+                    print e.args[0]
+                phedexNode = slc.localStageOut.get("phedex-node")
+                # If slc is correct, perform check
+                if "se-name" in slc.localStageOut and slc.localStageOut["se-name"] in node_map and phedexNode != None:
+                    assert phedexNode == node_map[slc.localStageOut["se-name"]], \
```

Comment:
Also please use the assert classes from unittest here, not the bare assert from python. self.assertEqual is probably what you want.